applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -6,7 +6,7 @@
 @import views.support.{RenderClasses, SeoOptimisedContentImage, StripHtmlTags, Video640}
 
 @defining(page.media, page.media match { case _: Audio => "audio" case _ => "video" }){ case (media, mediaType) =>
-<div class="l-side-margins l-side-margins--media">
+<div class="l-side-margins @if(!page.media.commercial.isAdvertisementFeature){l-side-margins--media}">
```

Comment:
Something is clearly going wrong if a class that ostensibly just fixes margins for media breaks GLabs skins.